config.json

```diff
@@ -8,6 +8,7 @@
     "hello-world",
     "anagram",
     "leap",
+    "gigasecond",
```

Comment:
Ahh, thanks for the reference. I didn't see the `exercises` list in the contributing guide or in other tracks (Ruby, Rust). I'll add that in.